src/test/java/htsjdk/samtools/reference/IndexedFastaSequenceFileTest.java

```diff
@@ -41,6 +41,7 @@
 public class IndexedFastaSequenceFileTest extends HtsjdkTest {
     private static File TEST_DATA_DIR = new File("src/test/resources/htsjdk/samtools/reference");
     private static File SEQUENCE_FILE = new File(TEST_DATA_DIR,"Homo_sapiens_assembly18.trimmed.fasta");
+    private static File SEQUENCE_FILE_GZ = new File(TEST_DATA_DIR,"Homo_sapiens_assembly18.trimmed.fasta.gz");
```

Comment:
I think this test file should be renamed to `AbstractIndexedFastaSequenceFileTest` now since it covers both.

Reply:
Done